input/pagecontent/RAD-Y1.md

```diff
@@ -133,37 +133,43 @@ There is a common set of attributes included in radiology diagnostic reports. Ta
+
+The Sender may include other renditions of the same report (e.g. PDF).
 
 For all rendered reports, the Sender shall set the DiagnosticReport.presentedForm.contentType with a value corresponding to the rendered report format.
 
 The Sender shall encode the rendered report that is referenced by DiagnosticReport.presentedForm in one of the following ways:
 - as a base64Binary in DiagnosticReport.presentedForm.data
-- as a base64Binary in a [Binary Resource](https://www.hl7.org/fhir/binary.html) and this Binary Resource is referenced in DiagnosticReport.presentedForm.url.
+- as a base64Binary in a [Binary Resource](https://www.hl7.org/fhir/binary.html), and this Binary Resource is referenced in DiagnosticReport.presentedForm.url.
   - The Binary Resource shall be in the Bundle. See FHIR Resolving references in Bundles at [http://hl7.org/fhir/bundle.html#references](http://hl7.org/fhir/bundle.html#references). 
 - host the rendered report somewhere and provide the url in DiagnosticReport.presentedForm.url
 
 The Sender shall populate accurate values for hash and size for the rendered report content in DiagnosticReport.presentedForm.hash and DiagnosticReport.presentedForm.size: 
 * Where the presentedForm is a Binary Resource instance, the .hash and .size shall represent the raw artifact that has been base64encoded in the Binary.data element.  
 * Where the presentedForm is hosted elsewhere, not as a Binary Resource, the .hash and the .size shall represent the rendered report content that would be retrieved using the mime-type specified in DiagnosticReport.presentedForm.contentType. 
 
-A Sender that supports the **PDF Report Option**, if configured, shall also attach a semantically equivalent diagnostic report in PDF format in the DiagnosticReport.presentedForm attribute. The presentedForm.contentType shall have the value "application/pdf". The Sender shall include in the PDF report all text and linked contents as in the HTML report. The Sender may preserve the linked contents as hyperlinks, or substitute the linked contents with the actual rendered contents and embedded them in the PDF file.
+A Sender that supports the **PDF Report Option**, if configured, shall also include a semantically equivalent diagnostic report in PDF format in the DiagnosticReport.presentedForm attribute. The presentedForm.contentType shall have the value "application/pdf". The Sender shall include in the PDF report all text and linked contents as in the HTML report. The Sender may preserve the linked contents as hyperlinks, or substitute the linked contents with the actual rendered contents and embedded them in the PDF file.
+
+###### TO DO: Create the right subsection heading (eg. Image References in Diagnostic Reports???) or, Find the right home for this content
+
+TO DO: Does this belong, instead, in a subsection under :4.Y1.4.1.2.3 IMR Observation Resource??
```

Comment:
No. This text belongs to the rendered report subsection. Basically it is saying that if the Observation has image references, then in the rendered report, it shall include a corresponding hyperlink.

I actually need to rework line 169 a bit because it does not quite work since the derivedForm only required to specify the study and series, but not instances. So it works for Report Reader but not Rendered Report Reader. I need to think about this.